packages/RNTester/.detoxrc.json

```diff
@@ -4,15 +4,15 @@
   "configurations": {
     "android.emu.debug": {
       "binaryPath": "android/app/build/outputs/apk/debug/app-debug.apk",
-      "build": "cd android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ..",
+      "build": "cd android && ./gradlew assembleDebug app:assembleAndroidTest -DtestBuildType=debug && cd ..",
       "type": "android.emulator",
       "device": {
-        "avdName": "testAVD"
+        "avdName": "Pixel_API_28"
```

Comment:
All looks good, just one concern. Will this work if the avdName is not 'Pixel_API_28' ?

Reply:
Yes, we'll have to change this to whatever our device is called I'm guessing.

Reply:
If that's the case, @chirag-singhal , can you please also add it in the readme ?
> Yes, we'll have to change this to whatever our device is called I'm guessing.

Reply:
@stealthanthrax there is a wiki link in readme which tells detailed steps to run tests on android. Should it be included in Readme as well?

Reply:
@chirag-singhal , wasn't aware of it. I would suggest that you mention it in the readme too 😅